src/transformers/dynamic_module_utils.py

```diff
@@ -198,7 +198,7 @@ def get_class_in_module(class_name: str, module_path: Union[str, os.PathLike]) -
     Returns:
         `typing.Type`: The class looked for.
     """
-    name = os.path.normpath(module_path).rstrip(".py").replace(os.path.sep, ".")
+    name = os.path.normpath(module_path).removesuffix(".py").replace(os.path.sep, ".")
```

Comment:
I get 

> AttributeError: 'str' object has no attribute 'removesuffix'

however (on python 3.8)

Reply:
Thank you for your review.
Oh, I failed to consider Python 3.8 support.